resources/views/admin/layouts/master.blade.php

```diff
@@ -62,12 +62,12 @@
 <script src="{{asset('bower_components/AdminLTE/dist/js/app.min.js')}}"></script>
 <script src="https://cdn.ckeditor.com/4.5.7/standard/ckeditor.js"></script>
 <script src="{{asset('bower_components/AdminLTE/plugins/timepicker/bootstrap-timepicker.min.js')}}"></script>
+<script src="{{asset('admin/js/main.js')}}"></script>
 <script>
```

Comment:
pls remove this javascript, we should not put the javascript code in layout view